src/chatgpt.ts

```diff
@@ -4,12 +4,17 @@ import { config } from "./config.js";
 import { execa } from "execa";
 import { Cache } from "./cache.js";
 import { ContactInterface, RoomInterface } from "wechaty/impls";
+import {string} from "yaml/dist/schema/common/string";
 
 const SINGLE_MESSAGE_MAX_SIZE = 500;
 export class ChatGPTBot {
   // Record talkid with conversation id
   conversations = new Map<string, ChatGPTConversation>();
   chatGPTPools: Array<ChatGPTAPI> | [] = [];
+  botConfig: { [key: string]: string } = {
```

Comment:
Maybe you could use `Record<string,string> `?

Reply:
botConfig: Record<string,string> = {
      trigger_keywords:""
  };